backend/core/src/shared/units-transformations.ts

```diff
@@ -205,20 +229,18 @@ const summarizeUnitsByType = (
     (unitType: string): UnitSummary => {
       const summary = {} as UnitSummary
       const unitsByType = units.filter((unit: Unit) => unit.unitType == unitType)
-      const finalSummary = Array.from(unitsByType).reduce((summary, unit) => {
-        return getUnitSummary(unit, summary)
+      const finalSummary = Array.from(unitsByType).reduce((summary, unit, index) => {
+        return getUnitsSummary(unit, index === 0 ? null : summary)
       }, summary)
-      if (finalSummary.minIncomeRange) {
-        finalSummary.minIncomeRange = minMaxInCurrency(finalSummary.minIncomeRange)
-      }
-      if (finalSummary.rentRange) {
-        finalSummary.rentRange = minMaxInCurrency(finalSummary.rentRange)
-      }
-
       return finalSummary
     }
   )
-  return summaries.filter((item) => Object.keys(item).length > 0)
+  return summaries.sort((a, b) => {
+    return (
+      UnitTypeSort.indexOf(a.unitType[0]) - UnitTypeSort.indexOf(b.unitType[0]) ||
```

Comment:
Isn't the value of unitType a string here, like "threeBdrm"? If that's the case, then a.unitType[0] would be "t" in this example.

Reply:
It's the index of the string in the array, so the value is a number

Reply:
Should be all set now 👍  I think the reason this typo didn't create any visible issues is because the income is also tied pretty deeply to unit size, and we don't have any units right now where unit would be a tie breaker for income